ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -734,30 +667,12 @@ case ${MatlabRunMode} in
 			interpreter=(octave-cli -q --no-window-system)
 		fi
 
-		if (( DoVol )); then
-			matlab_cmd="${ML_PATHS} fix_3_clean('${fixlist}',${aggressive},${MotionRegression},${AlreadyHP});"
-		else
-			matlab_cmd="${ML_PATHS} fix_3_clean('${fixlist}',${aggressive},${MotionRegression},${AlreadyHP},${DoVol});"
-		fi
+		matlab_cmd="${ML_PATHS} fix_3_clean('${fixlist}',${aggressive},${MotionRegression},${AlreadyHP},'${Caret7_Command}',${DoVol});"
 		
 		log_Msg "Run interpreted MATLAB/Octave (${interpreter[@]}) with command..."
 		log_Msg "${matlab_cmd}"
 		
-		# Use bash redirection ("here-string") to pass multiple commands into matlab
-		# (Necessary to protect the semicolons that separate matlab commands, which would otherwise
-		# get interpreted as separating different bash shell commands)
-		(
-			#fix's default settings.sh isn't safe to unbound variables or exit codes, so disable everything before sourcing
-			debug_disable_trap
-			set +u
-			source "${FSL_FIXDIR}/settings.sh"
-			set -u
-			debug_enable_trap
-			export FSL_FIX_WBC="${Caret7_Command}"
-			"${interpreter[@]}" <<<"${matlab_cmd}"
```

Comment:
Another instance where we've deleted the code, and not replaced it with anything, such that the revised version doesn't seem to be doing anything.

Reply:
Yes, good catch.